runtime/doc/debugger.txt

```diff
@@ -119,7 +119,6 @@ The following compile time preprocessor variables control the features:
     Debug Glyphs				FEAT_SIGNS
     Debug Highlights				FEAT_SIGNS
     Message Footer				FEAT_FOOTER
-    Balloon Evaluation				FEAT_BEVAL
```

Comment:
There's still a lot of references in the docs to this, but that should probably be dealt with elsewhere so I suppose it's fine.